src/plugins/intel_cpu/src/nodes/batch_to_space.cpp

```diff
@@ -157,6 +157,9 @@ void BatchToSpace::batchToSpaceKernel() {
     channels = channels == 0 ? 1 : channels;
     const size_t workAmount = inShape5D[0] * channels;
 
+    errorPrefix = "BatchToSpace layer with name '" + getName() + "'";
```

Comment:
It is already initialized inside the ctor. Why you need to do it one more time?

Reply:
No need to do that twice, fixed